pom.xml

```diff
@@ -23,11 +23,38 @@
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
```

Comment:
again not using most of it yet, plain spring-boot-starter or something like that would be enough

Reply:
I'm using web to test my data base